src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -374,6 +308,37 @@ private void ReloadCPython_Click(object sender, RoutedEventArgs e)
             dynViewModel.Model.OnRequestPythonReset(PythonServices.PythonEngineManager.CPython3EngineName);
         }
 
+        /// <summary>
+        /// This event is generated every time the user clicks a Radio Button in the Geometry Scaling section
+        /// The method just get the Radio Button clicked and saves the ScaleValue selected
+        /// This are the values used for the scales:
```

Comment:
`These` are

Reply:
comment deleted, check the next commit: https://github.com/DynamoDS/Dynamo/pull/13656/commits/789f51c4a3c106156940924585e1ca40abd9842b